src/vm/codeman.h

```diff
@@ -140,6 +140,10 @@ typedef struct _hpCodeHdr
     PTR_EE_ILEXCEPTION  phdrJitEHInfo;
     PTR_BYTE            phdrJitGCInfo;
 
+#if defined(FEATURE_GDBJIT)
+    VOID*            pCalledMethods;
```

Comment:
We need all targets. These thunks don't establish valid stack frame and break lldb unwinding. To help lldb we need to map native addresses to arbitrary symbolic names.